### PR TITLE
update some _id_attrs - cleanup for api 6.0

### LINKED ISSUE #3022: Double check the API 6.0 update if `_id_attrs` should have been updated

### PR DESCRIPTION
**Pull request by Bibo-Joshi**
closes #3022

### Checklist for PRs

- [x] Added `.. versionadded:: version`, `.. versionchanged:: version` or `.. deprecated:: version` to the docstrings for user facing changes (for methods/class descriptions, arguments and attributes)
- [x] Created new or adapted existing unit tests
- [ ] Documented code changes according to the [CSI standard](https://standards.mousepawmedia.com/en/stable/csi.html)
- [ ] Added myself alphabetically to `AUTHORS.rst` (optional)
- [ ] Added new classes & modules to the docs and all suitable `__all__` s

### CODE REVIEW

File: telegram/_inline/inlinekeyboardbutton.py [hunk 1]
```diff
@@ -62,6 +62,10 @@ class InlineKeyboardButton(TelegramObject):
 
         * After Bot API 6.1, only ``HTTPS`` links will be allowed in :paramref:`login_url`.
 
+    .. versionchanged:: 20.0
+       :attr:`web_app` is considered as well when comparing objects of this type in terms of
```
Comment (harshil21) [inline telegram/_inline/inlinekeyboardbutton.py:66]:
```suggestion
    .. versionchanged:: 20.0
        :attr:`web_app` is considered as well when comparing objects of this type in terms of
```
same in other places so it renders right

File: telegram/_keyboardbutton.py [hunk 2]
```diff
@@ -35,8 +35,8 @@ class KeyboardButton(TelegramObject):
     used instead of this object to specify text of the button.
 
     Objects of this class are comparable in terms of equality. Two objects of this class are
-    considered equal, if their :attr:`text`, :attr:`request_contact`, :attr:`request_location` and
-    :attr:`request_poll` are equal.
+    considered equal, if their :attr:`text`, :attr:`request_contact`, :attr:`request_location`,
+    :attr:`request_poll` and :attr:``web_app` are equal.
```
Comment (harshil21) [inline telegram/_keyboardbutton.py:39]:
```suggestion
    :attr:`request_poll` and :attr:`web_app` are equal.
```

File: telegram/_inline/inlinekeyboardbutton.py (hunk 1 above)
Comment (Bibo-Joshi) [reply telegram/_inline/inlinekeyboardbutton.py:66]: what about that doesn't render right for you? 🤔 this is how it looks like for me:

image

and the additional whitespace doesn't change anything about it …

Comment (harshil21) [reply telegram/_inline/inlinekeyboardbutton.py:66]: nvm, there was one sphinx version where that didn't render right

**Review by harshil21 — APPROVED** [review]
looks good!

**Pull request merged**